portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -25,18 +25,24 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns a comment*/
 @WebServlet("/new-comment")
 public class NewCommentServlet extends HttpServlet {
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("commentText");
     long timestamp_millis = System.currentTimeMillis();
-
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");
```

Comment:
Optional Nit: ISO date format is typically more readable for all audiences since it is unambiguous (between months and days). It is YYYY-MM-DD (or YYYY/MM/DD, or whatever).